docs/extending/admin_views.md

```diff
@@ -159,7 +159,7 @@ def index(request):
 def month(request):
     current_year = timezone.now().year
     current_month = timezone.now().month
-    calendar_html = calendar.HTMLCalendar().format_month(current_year, current_month)
+    calendar_html = calendar.HTMLCalendar().formatmonth(current_year, current_month)
```

Comment:
nice - thanks for finding this.
Looks like the correct fix to me https://docs.python.org/3/library/calendar.html#calendar.HTMLCalendar.formatmonth